ci/audit.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-node-go
+    tag: latest
+
+inputs:
+  - name: dp-developer-site
+    path: go/src/github.com/ONSdigital/dp-developer-site
```

Comment:
You can remove the path now that this repo has be converted to modules. You will need to do the same for the `build.yml` and `unit.yml`. See [this guide on converting to modules](https://github.com/ONSdigital/dp/blob/master/guides/MODULES.md) for more information.

Reply:
Have removed the inputs.path